sdk/communication/communication-sms/samples/v1/javascript/sendSms.js

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/**
+ * @summary Demonstrates how to use the SmsClient to send
+ * an SMS message
+ */
+
+const { SmsClient } = require("@azure/communication-sms");
+
+// Load the .env file if it exists
+const dotenv = require("dotenv");
+dotenv.config();
+
+export const main = async () => {
+  const connectionString =
+    process.env["COMMUNICATION_CONNECTION_STRING"] ||
+    "endpoint=https://<resource-name>.communication.azure.com/;<access-key>";
+  const client = new SmsClient(connectionString);
+  const sendResults = await client.send({
+    from: "<from-phone-number>",
+    to: ["<to-phone-number-1>", "<to-phone-number-2>"],
+    message: "Hello World via SMS!" // The message being sent
+  });
```

Comment:
Readability is pretty low here. Maybe add spaces between each line and annotate important lines. eg https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/keyvault/keyvault-keys/samples/typescript/src/cryptography.ts

Reply:
this one actually will be changed when i make this runnable because of the switch to env variables